exploring-the-universe/index.html

```diff
@@ -169,11 +169,12 @@ <h3>The story of humanity's journey through the cosmos</h3>
       <!-- This image is large making the LCP metric look bad on fresh load, which is the point.
            Additionally, we don't define a height, making the layout jump down once the browser
            knows the height of the image. This is so that the CLS score also looks bad. -->
-      <img src="galaxy.jpg" alt="The galaxy">
+      <img src="./images/galaxy.jpg" alt="The galaxy">
     </div>
   </header>
   <main>
     <section class="intro-section">
+      <p><a href="./README.md">Readme</a></p>
```

Comment:
should rendered page have Readme link?
would that load Readme in a good pane?
@captainbrosset

Reply:
No we should link to the github.com URL. The README.md does not get rendered as HTML on github.io, unless there's no index.html file present in the same repo.

```suggestion
      <p><a href="https://github.com/MicrosoftEdge/Demos/tree/main/exploring-the-universe">About this demo</a></p>
```